book/website/project-design/data-governance/data-gov-ml.md

```diff
@@ -0,0 +1,67 @@
+To address the issues of respecting data licenses, some ML teams are choosing to only use open source or permissively licensed data.
+This can be challenging to accomplish with data that does not have explicit licenses, or for data scraped for the web which may not have a license included in the metadata.
+
+### Governance Tool: Datasheets
+
+*We use the [Datasheets for Datasets](https://www.microsoft.com/en-us/research/uploads/prod/2019/01/1803.09010.pdf) from Microsoft in this chapter*.
+
+Datasheets are resources accompanying datasets that can provide key information about the motivation to create the dataset, 
+the data collection process, the dataset composition and pre-processing, and legal and ethical considerations when using the data.
+These resources helps contextualise the resource and is an opportunity for dataset creators to share any concerns, suggestions,
+and other advice regarding responsible use of the dataset.
+The Datasheets for Datasets paper contains a list of example questions that data creators should ask themselves and document.
+
+## Data Management
+
+Once data collection is complete, challenges emerge around data storage, access, and making changes.
+Because state-of-the-art ML datasets are very large, they tend to be stored in a central location, often a cloud provider database, and made available for download under certain circumstances.
+In addition to creating a governance structure for the opening or gating access to certain users, there is the challenge of creating a governance structure to allow people to request to opt-out their data from the dataset.
+Because opt-out occurs after a dataset has already been created, it will not remove the data from original versions of the dataset, but can facilitate removal from future versions.
+
+### Governance Tool: [Am I In the Stack?](https://huggingface.co/spaces/bigcode/in-the-stack)
+This tool was developed by the [BigCode](https://www.bigcode-project.org/) team to help developers inspect The Stack dataset so they cansee whether any of their repositories have been included and might be used for training ML models. 
+If that is the case, developers can fill out a form to request to opt-out from The Stack and have their data removed for future iterations of the dataset.
+
+### Governance Tool: [Have I Been Trained?](https://haveibeentrained.com/)
+This tool was developed by [Spawning](https://spawning.ai/) to help artists see if their work is a part of common datasets used to train AI art models and to opt-in or 
+opt-out to future training. They maintain a "registry of non-consenting data" where individuals and companies can check to see if their data is a part of these datasets.
```

Comment:
Here I'm still unclear on if the images are already under copyright (which makes it technically illegal) or if this is more a "CC license but not for ML" kind of situation.

Reply:
Not sure it's the right place to do a "call out" hahaha but I think it has been demonstrated that the data includes both images that are copyrighted and CC license with some use restrictions.

Not trying to delve into that issue too deeply here, but more to demonstrate that this is how orgs are thinking about how to improve transparency into what's in datasets.

Agree with you it's a big issue though.